mitodl/teachersportal
# Fixed snackbar dispatch

#### What are the relevant tickets?

Fixes #297 
#### What's this PR do?

Adds a check to prevent a recursive dispatch.
#### Where should the reviewer start?

App.js
#### How should this be manually tested?

Register an email locally. Upon successful registration you should see a snackbar pop up saying that registration was successful. If you click on the link in your email (this link will also be in the console) you should see an activation page which will immediately redirect to the page you were on before.
